contrib/download-frozen-image.sh

```diff
@@ -42,6 +42,8 @@ while [ $# -gt 0 ]; do
 	[ "$tag" != "$imageTag" ] || tag='latest'
 	tag="${tag%@*}"
 
+	imageFile="${image////_}" # "/" can't be in filenames :)
```

Comment:
FYI, OS X is still on bash 3 too; https://github.com/docker/compose/issues/1386#issuecomment-99198672